README.md

```diff
@@ -0,0 +1,5 @@
+## OIDC Client
+
+[![Build Status](https://travis-ci.org/CSCfi/oidc-client.svg?branch=master)](https://travis-ci.org/CSCfi/oidc-client)
+[![Coverage Status](https://coveralls.io/repos/github/CSCfi/oidc-client/badge.svg)](https://coveralls.io/github/CSCfi/oidc-client)
+[![Documentation Status](https://readthedocs.org/projects/csc-oidc-client/badge/?version=latest)](https://csc-oidc-client.readthedocs.io/en/latest/?badge=latest)
```

Comment:
```suggestion
[![Documentation Status](https://readthedocs.org/projects/csc-oidc-client/badge/?version=latest)](https://csc-oidc-client.readthedocs.io/en/latest/?badge=latest)

CSC OIDC Client is a lightweight [aiohttp](https://aiohttp.readthedocs.io/en/stable/) web application used for interacting with OIDC servers.
The source code is delivered with [ELIXIR AAI](https://elixir-europe.org/services/compute/aai) integration.
```

Reply:
also some installation instructions would be in order.

Reply:
Will do